salt/modules/win_psget.py

```diff
@@ -50,14 +49,42 @@ def __virtual__():
     return __virtualname__
 
 
-def _pshell(cmd, cwd=None, json_depth=2):
+def _ps_xml_to_dict(parent, dic=None):
+    '''
+    Formats powershell Xml to a dict.
+    Note: This _ps_xml_to_dict is not perfect with powershell Xml.
```

Comment:
@cmcmarrow 
What does this mean `is not perfect with powershell Xml`?

Reply:
bump @cmcmarrow

Reply:
XML and Dictionaries are not one to one. But in this uses cases it is. We should be using Json here but at the time there was a upstream bug in with the PowerShell program.

Reply:
@dwoz